README.md

```diff
@@ -166,8 +166,11 @@ Dates:
 
 Estimated cost of training: Equivalent of $7-15M
 
-Server training location: Ile-de-France, France
+Server training location: Île-de-France, France
 
+#### **Tokenization**
+The [BLOOM tokenizer](https://huggingface.co/bigscience/tokenizer) is a learned subword tokenizer trained using a byte-level Byte Pair Encoding (BPE) algorithm with byte fallback, a simple pre-tokenization rule, no normalization, and a vocabulary size of 250,680. This tokenizer was trained on a subset of a preliminary version of the corpus using alpha-weighting per language.    
```

Comment:
Just a small correction because our tokenizer does not use a fallback byte :hugs: 
```suggestion
The [BLOOM tokenizer](https://huggingface.co/bigscience/tokenizer) is a learned subword tokenizer trained using a byte-level Byte Pair Encoding (BPE) algorithm a simple pre-tokenization rule, no normalization, and a vocabulary size of 250,680. This tokenizer was trained on a subset of a preliminary version of the corpus using alpha-weighting per language.    
```

Reply:
Thank you! Sorry about that; I meant to remove it since you crossed it out in Slack, but it somehow fell through the cracks!